packages/neptune-extension/src/common/state/notebook/selectors.ts

```diff
@@ -3,3 +3,8 @@ import {AppState} from "common/state/reducers";
 export function getNotebookState(state: AppState) {
   return state.notebook;
 }
+
+export function getLoadingState(state: AppState) {
```

Comment:
This should probably be `getNotebookLoadingState` ?

Reply:
Yes